src/main/java/org/elasticsearch/index/percolator/stats/PercolateStats.java

```diff
@@ -152,11 +151,6 @@ public void readFrom(StreamInput in) throws IOException {
         percolateCount = in.readVLong();
         percolateTimeInMillis = in.readVLong();
         current = in.readVLong();
-        if (in.getVersion().before(Version.V_1_1_0)) {
-            in.readVLong();
-        } else {
-            in.readLong();
```

Comment:
not sure I follow you, why do we need reading and not using an always static value? Cant we just ditch it?